Fatal error starting logid

I am trying to get `logid` up and running on an Ubuntu 22.04 system with an MX Mechanical + MX Master 3S. I was able to pull down the latest (`be840b333`) and build / install with no issue, but the process core dumps on startup:

```
$ sudo systemctl start logid
$ sudo systemctl status logid
× logid.service - Logitech Configuration Daemon
     Loaded: loaded (/lib/systemd/system/logid.service; disabled; vendor preset: enabled)
     Active: failed (Result: core-dump) since Mon 2023-05-08 14:46:36 PDT; 14s ago
    Process: 8686 ExecStart=/usr/bin/logid (code=dumped, signal=ABRT)
   Main PID: 8686 (code=dumped, signal=ABRT)
        CPU: 6ms

May 08 14:46:35 onibaba systemd[1]: Started Logitech Configuration Daemon.
May 08 14:46:35 onibaba logid[8686]: [INFO] Config file does not exist, using empty config.
May 08 14:46:35 onibaba logid[8686]: [INFO] Detected receiver at /dev/hidraw7
May 08 14:46:36 onibaba logid[8686]: [ERROR] Orphan Receiver, missing DeviceManager
May 08 14:46:36 onibaba logid[8686]: [ERROR] Fatal error, file a bug report. Program will now exit.
May 08 14:46:36 onibaba logid[8686]: terminate called without an active exception
May 08 14:46:36 onibaba systemd[1]: logid.service: Main process exited, code=dumped, status=6/ABRT
May 08 14:46:36 onibaba systemd[1]: logid.service: Failed with result 'core-dump'.
```

So here I am filing a bug report. FWIW I have seen it enumerate the MX Mechanical in that list at least once, but never the MX Master. Anything worth trying in the config file?

--- a/src/logid/CMakeLists.txt
+++ b/src/logid/CMakeLists.txt
@@ -72,7 +72,6 @@ add_executable(logid
 set_target_properties(logid PROPERTIES RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
 
 pkg_check_modules(PC_EVDEV libevdev REQUIRED)
-pkg_check_modules(DBUS "dbus-1")
 pkg_check_modules(SYSTEMD "systemd")
 pkg_check_modules(LIBCONFIG libconfig REQUIRED)
 pkg_check_modules(LIBUDEV libudev REQUIRED)
@@ -111,13 +110,11 @@ elseif (NOT SYSTEMD_FOUND AND SYSTEMD_SERVICES_INSTALL_DIR)
     message(FATAL_ERROR "systemd is not found w/ pkg-config but SYSTEMD_SERVICES_INSTALL_DIR is defined.")
 endif ()
 
-if(DBUS_FOUND)
-    # Install DBus conf
-    # TODO: Is there a better way of setting the system policy directory?
-    set(DBUS_SYSTEM_POLICY_INSTALL_DIR "/usr/share/dbus-1/system.d")
-    configure_file(logiops-dbus.conf.in ${CMAKE_BINARY_DIR}/pizza.pixl.LogiOps.conf)
-    message(STATUS "dbus system policy will be installed at ${DBUS_SYSTEM_POLICY_INSTALL_DIR}")
-    install(FILES ${CMAKE_BINARY_DIR}/pizza.pixl.LogiOps.conf
-            DESTINATION ${DBUS_SYSTEM_POLICY_INSTALL_DIR}
-            COMPONENT cp)
-endif()
+# Install DBus conf
+# TODO: Is there a better way of setting the system policy directory?
+set(DBUS_SYSTEM_POLICY_INSTALL_DIR "/usr/share/dbus-1/system.d")
+configure_file(logiops-dbus.conf.in ${CMAKE_BINARY_DIR}/pizza.pixl.LogiOps.conf)
+message(STATUS "dbus system policy will be installed at ${DBUS_SYSTEM_POLICY_INSTALL_DIR}")
+install(FILES ${CMAKE_BINARY_DIR}/pizza.pixl.LogiOps.conf
+        DESTINATION ${DBUS_SYSTEM_POLICY_INSTALL_DIR}
+        COMPONENT cp)

